src/lib/formats/uef_cas.cpp

```diff
@@ -249,29 +249,31 @@ static int16_t* uef_cas_fill_bit( uint8_t loops, int16_t *buffer, bool bit )
 
 static int uef_cas_fill_wave( int16_t *buffer, int length, const uint8_t *bytes )
 {
+	const uint8_t *b = bytes;
```

Comment:
Isn’t this unnecessary?  `bytes` is `uint8_t const *`, not `uint8_t const *const` so you can change the value of the pointer, just not what it points to.